apps/dashboard/shared/components/design-system/avatars/ens-avatar/EnsAvatar.tsx

```diff
@@ -172,7 +182,7 @@ export const EnsAvatar = ({
           ) : (
             <span
               className={cn(
-                "text-primary text-sm",
+                "text-primary block max-w-full overflow-hidden truncate text-ellipsis whitespace-nowrap text-sm",
```

Comment:
hey yo, the `truncate` from tailwind already included: `overflow: hidden, white-space: nowrap, and text-overflow: ellipsis`
```suggestion
                 "text-primary truncate text-sm block"
```